backend/gn_module_export/admin.py

```diff
@@ -53,6 +56,61 @@ class ExportView(CruvedProtectedMixin, ModelView):
     module_code = "EXPORTS"
     object_code = None
 
+    def __init__(self, session, **kwargs):
+        # Référence au model utilisé
+        super(ExportView, self).__init__(Export, session, **kwargs)
+
+    def filter_user_app_and_role():
+        id_app_gn = (
+            Application.query.with_entities(Application.id_application)
+            .filter(Application.code_application == "GN")
+            .first()
+        )
+
+        user_and_gp_from_gn_app = (
+            User.query.join(
+                CorRole, User.id_role == CorRole.id_role_utilisateur, isouter=True
+            )
+            .join(
```

Comment:
On utilise plutôt directement `outerjoin` : 
```suggestion
            .outerjoin(
```